### PR TITLE
Configure GitHub native Dependabot for dependency management

### PR DESCRIPTION
**Pull request by KeisukeYamashita**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

/kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area build

> /area config

> /area outputs

> /area tests

**What this PR does / why we need it**:

This PR will configure GitHub native Dependabot.

We now support many kinds of outputs using many external packages.
To improve security and dependency management, it's better to introduce Dependabot.

Dependabot supports not only Go modules but also Docker base images.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

## Ref

* [Keep all your packages up to date with Dependabot, GitHub Blog](https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/)
* [Enabling and disabling version updates, GitHub Doc](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/enabling-and-disabling-version-updates#enabling-github-dependabot-version-updates)

### CODE REVIEW

**Comment by Issif** [comment]
That's a good idea, I never used until now so I need to read docs before. I'm also wondering if we want a global method for the whole organization. cc @leogr @leodido @fntlnz @nibalizer

**Comment by Issif** [comment]
@KeisukeYamashita As this PR may break CI, I would like to postpone it to 3.0.0, wdyt? It's the onyl one which remains for releasing 2.20.0 and I would like to to so for new year.

**Comment by KeisukeYamashita** [comment]
@Issif 

> As this PR may break CI, I would like to postpone it to 3.0.0, wdyt? 

Okay👍　I will close this PR and reopen when we need it. 

Anyway, it will be difficult to maintain dependency without a dependency management tool. There are many options (dependabot, renovate, etc) and Dependabot is not the only option. We can decide later. 

Thank you!😆

**Comment by Issif** [comment]
I reopened it, and changed milestone and project for upcoming 3.0.0 release. We'll discuss about in community meeting to have a global feeling

/hold

**Comment by leogr** [comment]
@Issif @KeisukeYamashita 
Any update on this?

**Comment by KeisukeYamashita** [comment]
Version 2 is still stable & latest version from GitHub so still works 👍  Let's see what @Issif has in mind 😄

**Comment by Issif** [comment]
I agree we should upgrade up to Go 1.17 first. For dependabot, if you feel confident, go ahead, I was thinking to add it in v3 of falcosidekick, with the complete refactorization of the repo structures (/cmd, /internal, etc)

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: docker
+    directory: "/"
+    schedule:
+      interval: daily
+  - package-ecosystem: gomod
+    directory: "/"
+    schedule:
+      interval: daily
```
Comment (cpanato) [inline .github/dependabot.yml:10]: would add a limit of prs per day

```
open-pull-requests-limit: 10
```

Comment (Issif) [reply .github/dependabot.yml:10]: @KeisukeYamashita can you update this, please? I'll approve your PR after. thanks

**Pull request closed without merging**